src/test/java/org/jitsi/meet/test/DesktopSharingTest.java

```diff
@@ -134,6 +129,9 @@ private void checkExpandingDesktopSharingLargeVideo(boolean isScreenSharing)
         MeetUIUtils.clickOnToolbarButton(
             getParticipant1().getDriver(), "toggleFilmstripButton");
 
-        TestUtils.waitMillis(5000);
+        TestUtils.waitForDisplayedElementByXPath(
+            getParticipant1().getDriver(),
+            "//span[@id='localVideoWrapper']",
```

Comment:
this duplicates the constant and could be a TestUtil method, but I give up since you want to merge it now